platforms/shared/configuration/setup-k8s-environment.yaml

```diff
@@ -100,6 +100,15 @@
     when: 
     - network.type == 'fabric'
     - network.env.type == 'operator'
+
+  # Deploy prometheus server
+  - name: Deploy prometheus server
+    include_role:
+      name: setup/prometheus
+    vars:
+      prometheus_port: "{{ network.organizations[0].prometheus.port }}"
```

Comment:
better to have Prometheus check and port at network level. and use http/https ports for Prometheus

Reply:
Fixed.